.github/workflows/ci.yml

```diff
@@ -321,10 +321,6 @@ jobs:
             profiles: all-tas-tests,elastic
             search-engine-type: elasticsearch
             db-type: mariadb
-          - testSuite: Elasticsearch MS SQL
-            profiles: all-tas-tests,elastic
-            search-engine-type: elasticsearch
-            db-type: mssql
           - testSuite: Elasticsearch Oracle 19.3.0-ee
```

Comment:
Can we just comment the MS SQL matrix section out instead of deleting it, and add a reference in the comment to https://alfresco.atlassian.net/browse/ACS-6177?

Reply:
Done